<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="maximumSize">
       <size>
        <width>126</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>入库</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出库</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>转挂</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>库存查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>操作记录查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>物料信息查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新增物料编号</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>备注</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>入库</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>挂账人</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string>1</string>
           </property>
           <property name="background">
            <color alpha="0">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </property>
          </row>
          <column>
           <property name="text">
            <string>物料编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物料名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>操作</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxProvince"/>
          </item>
          <item row="3" column="11">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>出库</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBoxCity"/>
          </item>
          <item row="1" column="7">
           <widget class="QComboBox" name="comboBoxCommunity"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>区/县</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>社区</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>小区</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>省</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBoxTown"/>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="comboBoxCounty"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>市</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QComboBox" name="comboBoxVillage"/>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>街道</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>备注</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="5">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="1" column="11">
           <widget class="QPushButton" name="pushButtonAddCommunity">
            <property name="text">
             <string>新增小区</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>物料编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物料名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>操作</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox_9"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>备注</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>转入人</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_8"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>转出人</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>转挂</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_3">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>物料编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>物料名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>操作</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="8">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>库存查询</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="0" column="6">
           <widget class="QComboBox" name="comboBox_10">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>物料编号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>挂账人</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>物料名称</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="2">
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2021</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>截止时间</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>起始时间</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="comboBox_12"/>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>操作人</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QDateTimeEdit" name="dateTimeEdit_2">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2021</year>
              <month>12</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="date">
             <date>
              <year>2021</year>
              <month>12</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>操作类型</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBox_11"/>
          </item>
          <item row="2" column="6">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>操作记录查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>物料名称</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>物料信息查询</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>物料编号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="4">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>./default.xlsx</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_8">
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_7">
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>新增物料编号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_4"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionChangePassword"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>用户登录</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>用户密码修改</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
